samples/spm/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: LicenseRef-Nordic-5-Clause */
+
+/ {
+	chosen {
+		zephyr,console=&spm_uart;
+	};
+};
+
+&uart0 {
+	status = "okay";
+
+	spm_uart: nrf-spm-uart {
```

Comment:
Should we have a comment here explaining why this is needed? Also, is this needed if non secure logs should not be dumped?